src/open_inwoner/utils/templatetags/utils.py

```diff
@@ -98,3 +100,8 @@ def readable_size(value):
 @register.filter
 def cookies_accepted(request: HttpRequest) -> bool:
     return request.COOKIES.get("cookieBannerAccepted") == "true"
+
+
+@register.simple_tag
+def warning_banner_enabled() -> bool:
```

Comment:
Don't do this via a templatetag, they are already overused in the project. See utils/context_processors.py